crates/stages/src/stages/hashing_account.rs

```diff
@@ -293,6 +252,21 @@ impl<DB: Database> Stage<DB> for AccountHashingStage {
     }
 }
 
+/// Flushes channels hashes to ETL collector.
+fn collect(
+    channels: &mut Vec<Receiver<(RawKey<B256>, RawValue<Account>)>>,
+    collector: &mut Collector<RawKey<B256>, RawValue<Account>>,
+) -> Result<(), StageError> {
+    for channel in channels.iter_mut() {
+        while let Ok((key, v)) = channel.recv() {
+            collector.insert(key, v)?;
+        }
+    }
+    debug!(target: "sync::stages::hashing_account", "Hashed {} entries", collector.len());
```

Comment:
`collector.len()` contains number of entries that's currently in the collector, is it guaranteed that all entries from the previous calls of `collect` are flushed?

Reply:
len() contains number of entries that got inserted during its whole lifetime, not just the ones in memory